### DISCUSSION TITLE
Grimoire Lab/MacOS M2

### DISCUSSION THREAD

**Discussion by devaniranjan** [chaoss/grimoirelab#673] [Q&A] [answered]
Has anyone had success installing grimoirelab using the instructions here?
https://github.com/chaoss/grimoirelab/blob/master/docker-compose/README.md#mac

I have tried bug fixing it but docker-compose up command fails.

**Comment by jjmerchante** [comment, marked as answer]
Hi @devaniranjan ,

The docker-compose command executes the default `docker-compose.yml` file. Mac users have reported issues with the Kibana versions provided in that file. I recommend using OpenSearch Dashboards instead. To do this, run
```
docker-compose up -d -f docker-compose-opensearch.yml
```
which contains images compatible with the new Mac chipsets.

Currently, this setup does not automatically import dashboards. You need to import the dashboards you want or create your own manually. You can find some dashboards at the following link: [OpenSearch Dashboards JSON files](https://github.com/chaoss/grimoirelab-sigils/tree/master/panels/json/opensearch_dashboards).

To import these dashboards into OpenSearch Dashboards you need to open the menu, select "Stack Management", look for "Saved Objects" and click on the "Import" button.
